docs/development.md

```diff
@@ -92,84 +93,143 @@ $ boilerplate/_lib/container-make
 
 ## How to run
 
-Regardless of how you choose to run the operator, before doing so you will need to install the `UpgradeConfig` CRD on your cluster:
+Regardless of how you choose to run the operator, before doing so ensure the `UpgradeConfig` CRD is present on your cluster:
 
 ```shell
 $ oc create -f deploy/crds/upgrade.managed.openshift.io_upgradeconfigs_crd.yaml
 ```
 
 ### Locally
 
-- Make sure you have the [operator-sdk](https://github.com/operator-framework/operator-sdk/releases) binary in your `$PATH`.
+- Make sure you have the [operator-sdk](#operator-sdk) `$PATH`.
 
 - If you are not using an account that has `cluster-admin` privileges, you will need to [elevate permissions](https://github.com/openshift/ops-sop/blob/master/v4/knowledge_base/manage-privileges.md) to possess them.
 
-- Create a project for the operator to run inside of.
+- Create a project for the operator to run inside of:
 
 ```
-$ oc new-project managed-upgrade-operator
+$ oc new-project test-managed-upgrade-operator
 ```
 
 - Run the operator via the Operator SDK:
 
 ```
-$ OPERATOR_NAMESPACE=managed-upgrade-operator operator-sdk run --local --watch-namespace=""
+$ OPERATOR_NAMESPACE=test-managed-upgrade-operator operator-sdk run --local --watch-namespace=""
 ```
 
 (`make run` will also work here)
 
-- Trigger a reconcile loop by applying an `upgradeconfig` CR with your desired specs.
+- Trigger a reconcile loop by applying an `upgradeconfig` CR with your desired specs:
 
 ```shell
 $ oc apply -f test/deploy/upgrade.managed.openshift.io_v1alpha1_upgradeconfig_cr.yaml
 ```
 
 ### Remotely
 
-- Create a project for the operator to run inside of.
+- Build the image. In this example, we will use [Quay](http://quay.io/) as the container registry for our image:
 
 ```shell
-$ oc new-project managed-upgrade-operator
+$ operator-sdk build quay.io/<QUAY_USERNAME>/managed-upgrade-operator:latest
 ```
 
-- Build the image. In this example, we will use [Quay](http://quay.io/) as the container registry for our image.
+- Push the image to the registry:
+
+---
+
+**NOTE**
```

Comment:
The note here split the topic into two parts, makes me a bit confusing when reading.

Maybe put the note to the bottom of the whole Remotely topic?